pkgs/applications/networking/mailreaders/meli/default.nix

```
@@ -0,0 +1,50 @@
+{ stdenv
+, lib
+, fetchgit
+, rustPlatform
+, pkgconfig
+, openssl
+, dbus
+, sqlite
+, file
+, gzip
+, notmuch
+, withNotmuch ? true
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "meli";
+  version = "alpha-0.5.1";
+
+  src = fetchgit {
+    url = "https://git.meli.delivery/meli/meli.git";
+    rev = "${version}";
+    sha256 = "1y5567hdm1s2s272drxvmp6x4y1jpyl7423iz58hgqcsjm9085zv";
+  };
+
+  cargoSha256 = "040dfr09bg5z5pn68dy323hcppd599d3f6k7zxqw5f8n4whnlc9y";
+
+  cargoBuildFlags = [ (lib.optionalString withNotmuch "--features=notmuch") ];
```

```suggestion
  cargoBuildFlags = lib.optional withNotmuch "--features=notmuch";
```
ref: https://github.com/NixOS/nixpkgs/blob/master/lib/lists.nix#L242-L254